templates/core/about/metadata.html

```diff
@@ -14,7 +14,7 @@ <h1>Metadata for custom builds</h1>
 
 	<p>The available configuration flags you can customize are:</p>
 
-	{% filter highlight(lang="toml") %}
+	{% filter highlight("toml") %}
```

Comment:
something seems off with this page: 
<img width="948" alt="grafik" src="https://github.com/rust-lang/docs.rs/assets/540890/67f7481d-0960-4972-a828-4f8bce15f108">

Reply:
Ah yes, forgot `|safe`.

Reply:
@GuillaumeGomez still looks like this

Reply:
Indeed, might be a bug in rinja. Investigating!

Reply:
Fixed. :)